src/components/progress/progress.scss

```diff
@@ -0,0 +1,149 @@
+@import "../../themes/ionic.globals";
+
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:               rgb(16, 108, 2001) !default;
```

Comment:
Hmmm, yes you are right. It would probably be good to at least make the `progress.mode.scss` files for determining the color of the progress bar per mode, and then using our color functions for the primary and buffer color.

In order for this to work though we need to change some things in the Progress class so that the color classes will be added. This causes other issues because of the `progress` class in the template. I'm thinking we should rename the template classes to be more specific (also makes it less likely to have conflicts with user's classes).

I created a commit for these changes here: https://github.com/ionic-team/ionic/commit/cfb39726698952e967be29253815528c02becf00

Naming is up for discussion if any of them don't make sense, but it reduces nesting as well in the Sass files. I'm sure some improvements can be made there, let me know your thoughts.

@realappie Are you on Slack? If you send me or comment your email I could invite you to ours for discussion.